Latex-Briefvorlage.tex

```diff
@@ -114,8 +106,30 @@
 
 % Ihr Schreiben vom
 %\setkomavar{yourmail}[\yourmailname]{1. April 2018}
+% -------------------------------
+
+% Metadaten in PDF schreiben
+\begingroup
```

Comment:
Einer der Hauptgründe für die Erstellung dieser Latex-Vorlage war, dass die meisten, die ich gefunden hatte zu kompliziert waren. Diese Vorlage ist für Leute, die 1-2 Mal im Jahr einen Brief schreiben müssen und bis zu nächsten mal alle fast alle Latex-Details wieder vergessen haben. Ich habe hier bei dem Block die Befürchtung, dass dieser das Verständnis erschwert und daher "zu kompliziert" ist.

Reply:
Hier muss nichts zwingend geändert werden - der Betreff und der Absender werden automatisch als Metadaten in die PDF-Datei geschrieben. Aber ich kann verstehen, wenn das mehr zu Verwirrung führt als dass es hilft.